preview/app/http/PreviewContentSecurityPolicyFilter.scala

```diff
@@ -0,0 +1,19 @@
+package http
+
+import org.apache.pekko.stream.Materializer
+import play.api.mvc.{Filter, RequestHeader, Result}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+// This should be kept up to date with the prod configured in VCL (fastly-edge-cache repo).
+// We don't have fastly or a cache in front of `preview.gutools.co.uk` as it provisioned in CloudFormation in the `platform` repo (private):
+// https://github.com/guardian/platform/blob/main/provisioning/cloudformation/frontend.yaml#L824
```

Comment:
If pointing to a specific line, should we make this a permalink?
```suggestion
// https://github.com/guardian/platform/blob/06c343b0c44ba5bdcd1d7b613aed5e896f62684a/provisioning/cloudformation/frontend.yaml#L824
```